internal/api/contract/OrgFeatureFlagResponse.go

```diff
@@ -0,0 +1,8 @@
+package contract
+
+type OrgFeatureFlagResponse struct {
+	Ok          bool   `json:"ok,omitempty"`
+	UserMessage string `json:"userMessage,omitempty"`
+	Code        string `json:"code,omitempty"`
```

Comment:
**Question:** Did you consider unmarshaling Code into an `int`? This would mean we could switch our checks to use the `http.StatusUnauthorized` and `http.StatusForbidden` constants. 

```suggestion
	Code        int `json:"code,omitempty,string"`
```

Reply:
The API provides a String, using an Integer makes the unmarshalling fail.

Reply:
`{"code":"401","status":"Unauthorized"}`